e2e/protractor/suites/search/search-results-files-folders.test.ts

```diff
@@ -92,7 +92,7 @@ describe('Search results - files and folders', () => {
     await dataTable.waitForBody();
 
     const fileEntry = await apis.user.nodes.getNodeById(fileId);
-    const modifiedDate = moment(fileEntry.entry.modifiedAt).format('MMM D, YYYY, h:mm:ss A');
+    const modifiedDate = format(new Date(fileEntry.entry.modifiedAt), 'MMM d, yyyy, h:mm:ss A');
```

Comment:
Please check the 'fileEntry.entry.modifiedAt' type & use the format according to that.

Reply:
Checked and done.